projects/cypress/CYW954907AEVAL1F/wicedstudio/aws_tests/.project

```diff
@@ -637,9 +637,14 @@
 			<locationURI>AWS_IOT_MCU_ROOT/libraries/freertos_plus/aws/greengrass/include/aws_greengrass_discovery.h</locationURI>
 		</link>
 		<link>
-			<name>libraries/c_sdk/aws/defender/test/aws_iot_tests_defender_api.c</name>
+			<name>libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c</name>
 			<type>1</type>
-			<locationURI>AWS_IOT_MCU_ROOT/libraries/c_sdk/aws/defender/test/aws_iot_tests_defender_api.c</locationURI>
+			<locationURI>AWS_IOT_MCU_ROOT/libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c</locationURI>
+		</link>
+		<link>
+			<name>libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c</name>
+			<type>1</type>
+			<locationURI>AWS_IOT_MCU_ROOT/libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c</locationURI>
```

Comment:
Seems like the .project files that were updated via script also added an entry for `libraries/3rdparty/tinycbor/cborpretty_stdio.c`. Does the same need to be done for cypress and espressif?

Reply:
Yes I think so since cborpretty_stdio.c is a dependency for the defender system tests. I've updated the .project files for cypress and espressif: cacbd1cc & eede657e7